### ISSUE TITLE
Ycb video dataset

### ISSUE THREAD

**Issue by MinWang1997** [PyTorchLightning/pytorch-lightning#10250]
## 🚀 Feature

add ycb video dataset to vision data modules 

### Motivation
I noticed that the CVPR 2021 oral paper FFB6D used test data as validation set and test dataset…
I'm always frustrated when I deal with data preprossing and dataloader.

### Pitch

Create standard ycb dataset as preßbuilt datamodules for 6D pose estimation.

**Comment by awaelchli** [comment]
Hello @MinWang1997 
We don't add dataset templates to Lightning. Did you mean to submit this issue to a different repository?

**Comment by MinWang1997** [comment]
> Did you mean to submit this issue to a different repository?

No, I mean I do not know how to convert a dataset and its complex preprocessing and get item step in a repository to lighting data module. Could you give me a general template?

**Comment by awaelchli** [comment]
Hey @MinWang1997 
The DataModule is not a special dataset. You can 100% reuse the dataset code (e.g. ycb dataset) in a datamodule The datamodule only groups data related code, like splitting the dataset or grouping hyperparameters for the preprocessing. 

See here a simple example of a DataModule: 
https://pytorch-lightning.readthedocs.io/en/latest/extensions/datamodules.html#what-is-a-datamodule

To get started, copy the code example there and replace MNIST with YCBDataset. 

Hope this helps.
